src/Microsoft.ML.TorchSharp/NasBert/NasBertTrainer.cs

```diff
@@ -583,13 +584,13 @@ private IList<int> PrepInputTokens(ref ReadOnlyMemory<char> sentence1, ref ReadO
                 getSentence1(ref sentence1);
                 if (getSentence2 == default)
                 {
-                    return new[] { 0 /* InitToken */ }.Concat(tokenizer.EncodeToConverted(sentence1.ToString())).ToList();
+                    return InitTokenArray.Concat(tokenizer.EncodeToConverted(sentence1.ToString())).ToList();
```

Comment:
>return InitTokenArray.Concat(tokenizer.EncodeToConverted(sentence1.ToString())).ToList(); [](http://example.com/codeflow?start=20&length=89)

nit:

If you want to save one more allocation here, you may do something like:

```C#
List<int> newList = new List<int>(tokenizer.EncodeToConverted(sentence1.ToString()));
newList.Insert(0, 0);
return newList;
```

You don't have to do it if it is not worth it.

Reply:
Changed it to that.